docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+{% extends "invenio_administration/create.html" %}
+<!-- Dashboard page -->
+{% extends "invenio_administration/index.html" %}
+```
+
+In each page we can override or extend different blocks, mainly there are 2 blocks that can be overrided to display our custom content:
+
+- `admin_page_content`: Main content each page. It's present in all the views.
+- `javascript`: Main block to place any javascript content. It's important to always use {{ super() }} inside this block to inherit all the javascript needed to render the rest of the page.
+
+More information about `blocks` is available in [Jinja documentation](https://jinja.palletsprojects.com/en/3.0.x/templates).
+To find more about the different blocks that can be overridden in each of the views you can check the code of the views [here](https://github.com/inveniosoftware/invenio-administration/tree/main/invenio_administration/templates/semantic-ui/invenio_administration).
+
+### Customisation: React components
+
+In addition to jinja, in each view there is the possibility to inject React components. In order to do this, the view must be extended with a custom jinja template as described above. In this template, the javascript code that will render the view should be declared in the `javascript` block and in this block, the file that will render the React components must be included in webpack. In `webpack.py` the following line must be added:
+
+```diff
+theme = WebpackThemeBundle(
+    __name__,
+    "assets",
+    default="semantic-ui",
+    themes={
+        "semantic-ui": dict(
+            entry={
++               "invenio-administration-search":
++                   "./js/invenio_app_rdm/src/search/search.js",
+                ...
+```
+
+Once the file was added to webpack we can add it to the template we are overriding. The webpack configuration **must** be registered in the module entry point!
+
+```html
+{% extends "invenio_administration/details.html" %}
+
+{% block admin_page_content %}
+  <div
+    id="invenio-details-config"
+  >
+  </div>
+{% endblock admin_page_content %}
+{% block javascript %}
+  {{ super() }}  # don't forget the parent JS assets!
+  {{ webpack['invenio-administration-search.js'] }}  # it is crucial to remember about adding proper assets to your jinja 
+{% endblock %}
+```
+
+When overriding the views, the block `admin_page_content` can be overridden, and a new `div` with a custom id can be set so that, in the new javascript file that was added to webpack the components can be injected in the view by looking for the correct id in the DOM and, like in the following example:
+
+```javascript
+const domContainer = document.getElementById("invenio-details-config");
+domContainer &&
+  ReactDOM.render(
+    <MyCustomComponent />,
+    domContainer
+  );
+
+```
+
+For most of the views there is no need to override the `admin_page_content` block as the React component can be injected in the already existing view - by reusing existing DOM and React root element IDs 
```

Comment:
I think that here we want to say that, if you need to change only the React part, you don't need to override the `admin_page_content` block. Is this correct?
If yes, we might want to rephrase a bit.